src/libraries/math/NewFullMath.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: UNLICENCED
```

Comment:
Where is this file coming from? Is it not possible to install it as a forge-install dependency?

Reply:
It's coming from https://github.com/Uniswap/v3-core whose contracts all have pragmas which are not compatible with molten's.

https://github.com/Uniswap/v3-core/blob/main/contracts/libraries/Oracle.sol
https://github.com/Uniswap/v3-core/blob/main/contracts/libraries/FullMath.sol
https://github.com/Uniswap/v3-core/blob/main/contracts/libraries/TickMath.sol